wild_align:
  # WiLD topics 
  numberRobots: 2                                                           #determines the indexing of robot data
  rangeTopics: ["wild_align/range1"]                                           #numberRobots-1 topics for sub range data
  keyframeTopics: ["wild_align/local_keyframe","wild_align/peer1_keyframe"] #numberRobots topics for sub keyframe data
  pathTopics: ["wild_align/local_path","wild_align/peer1_path"]             #numberRobots topics for pub path data   
  cloudTopics: ["wild_align/local_cloud","wild_align/peer1_cloud"]             #numberRobots topics for pub cloud data  
  keyframePublishTopic: "wild_align/pub_keyframe"                           #topic for releasing keyframes
  relativeMeasurementsTopic: "wild_align/relative_measurements" 

  # WiLD variables
  publishFrequency: 1               #Rate for publishing thread       
  requestFrequency: 1.5             #Rate for sending data requests
  searchFrequency: 0.5              #Rate for searching for loop closure candidates

  reistrationSearchDistance: 5.0    #How close should keyframes be for reliable loop closure
  rangeNoiseBound: 7.0              #Reasonable error bound for range measurements 
  use2Dmapping: true                #Selects the 2D or 3D range solver
  trustRSSIThreshold: 120           #Number of RSSI measurements to be collected for a reliable model (referenced in paper)

  residualErrorThresh: 0.3          #ICP residual threshold
  bowSearchWindow: 3                #Size of keyframe neighborhood to be searched via BoW
  

  #Demo variables activate normally unhelpful behaviors for visualizations for 2 robots
  demoMode: true                   #Share all point clouds for visualization
  reportData: true                 #Report size of data recieved 
  verbose: true                              
  saveMap: false    




